anchore_engine/services/policy_engine/__init__.py

```diff
@@ -350,6 +352,33 @@ def handle_feed_sync_trigger(*args, **kwargs):
     return True
 
 
+def handle_grypedb_sync(*args, **kwargs):
+    """
+    Calls function to run GrypeDBSyncTask
+
+    :param args:
+    :param kwargs:
+    :return:
+    """
+    system_user = _system_creds()
+    if "GrypeDBSyncTask" not in locals():
```

Comment:
This looks surprising... why is the use of `locals()` needed?

Reply:
I did this becuase I was copying the approach used by the other monitor handlers in the file. I spoke to @zhill and this is a holdover from a time when there were issues with how code was being imported. I removed the conditional check but still import in the function since this is the init file